101-deploy-dsc-pullserver-to-win-server/azuredeploy.json

```diff
@@ -0,0 +1,304 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "StorageName": {
+            "type": "string",
+            "defaultValue": "pctst03stor",
+            "metadata": {
+                "description": "The name of the new storage account created to store the VMs disks"
+            }
+        },
+        "StorageType": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS"
+            ],
+            "metadata": {
+                "description": "The storage type created to store the VMs disks"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "West Europe",
+            "allowedValues": [
+                "East US",
+                "West US",
+                "West Europe",
+                "East Asia",
+                "South East Asia"
+            ],
+            "metadata": {
+                "description": "The location for the vms and other artefacts to be created."
+            }
+        },
+        "AdminUserName": {
+            "type": "string",
+            "defaultValue": "domadm",
+            "metadata": {
+                "description": "The admin user name for the VM created."
+            }
+        },
+        "AdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The admin password for the VM created."
+            }
+        },
+        "WindowsOSVersion": {
+            "type": "string",
+            "defaultValue": "2012-R2-Datacenter",
+            "allowedValues": [
+                "2008-R2-SP1",
+                "2012-Datacenter",
+                "2012-R2-Datacenter",
+                "Windows-Server-Technical-Preview"
+            ],
+            "metadata": {
+                "description": "The version of the Windows OS for the VM."
+            }
+        },
+        "ImagePublisher": {
```

Comment:
This can be a hardcoded variable as the template scenario depends on a version of Windows Server. You don't want the user to specify a different publisher than MicrosoftWindowsServer